vonk/plugins/plugins_configclasses.rst

```diff
@@ -7,7 +7,7 @@ A configuration class is a static class with two public static methods having th
 
 .. code-block:: csharp
 
-   [VonkConfiguration (order: xyz)] //xyz is an integer
```

Comment:
The class is called `VonkConfigurationAttribute`, but then you use it as `[VonkConfiguration]`. This is just how attributes in C# work...